lib/backend/k8s/conversion/conversion.go

```diff
@@ -555,7 +573,72 @@ func (c Converter) SplitNetworkPolicyRevision(rev string) (crdNPRev string, k8sN
 		return
 	}
 
-	crdNPRev = revs[0]
-	k8sNPRev = revs[1]
+	RevA = revs[0]
+	RevB = revs[1]
+	return
+}
+
+// serviceAccountNameToProfileName creates a profile name that is a join
+// of 'ksa.' + namespace + "." + serviceaccount name.
+func serviceAccountNameToProfileName(sa, namespace string) string {
+	// Need to incorporate the namespace into the name of the sa based profile
+	// to make them globally unique
+	if namespace == "" {
```

Comment:
Is there a particular case where this is expected?

I think that the namespace will _always_ be populated (even in the "default" case).

This code doesn't hurt, but wondering if there was a specific reason it was added, or just being safe.